src/systemd/sd-dhcp6-client.h

```diff
@@ -184,8 +184,14 @@ enum {
         SD_DHCP6_OPTION_SLAP_QUAD                  = 140, /* RFC 8948 */
         SD_DHCP6_OPTION_V6_DOTS_RI                 = 141, /* RFC 8973 */
         SD_DHCP6_OPTION_V6_DOTS_ADDRESS            = 142, /* RFC 8973 */
-        SD_DHCP6_OPTION_IPV6_ADDRESS_ANDSF         = 143 /* RFC 6153 */
+        SD_DHCP6_OPTION_IPV6_ADDRESS_ANDSF         = 143, /* RFC 6153 */
         /* option codes 144-65535 are unassigned */
+        /* DHCPv6 private options. Assuming private options base as 224
+         * derived from DHCPv4 and end as 65535
+         */
+        _SD_DHCP6_OPTION_PRIVATE_BASE               = 224,
+        _SD_DHCP6_OPTION_PRIVATE_LAST               = 65534,
+        _SD_DHCP6_OPTION_END                        = 65535
```

Comment:
add trailing comma

Reply:
Added trailing comma, but if I understand correctly it is not mandatory to have trailing comma at the end of enum. correct me if i am wrong.

Reply:
actually this is a public header (or treated as such) so it's compiled against weird combinations, and some of them reject trailing commas in enums

Reply:
Yeah, sorry for the noise.

Reply:
It failed in the below test case.
https://github.com/systemd/systemd/actions/runs/6049185778/job/16415989588